.github/workflows/build.yml

```
@@ -2,11 +2,14 @@ name: Build UI and Go App
 
 on:
   push:
+    branches: [main]
     branches: [main]
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z0-9]+"
   pull_request:
     branches: [main]
+    branches: [main]
```

actionlint (1.7.8)

key "branches" is duplicated in "pull_request" section. previously defined at line:11,col:5

(syntax-check)